### PR TITLE
merge e2e profile test project

### PR DESCRIPTION
**Pull request by mrproliu**
…orage types

Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [ ] New feature provided
- [x] Improve performance

- Related issues
#4104 

___
### New feature or improvement
- The current e2e test cases for profile use different storage to distinguish different projects. Now merge the test projects into one. Use `env` to distinguish different storage.

### CODE REVIEW

**Comment by mrproliu** [comment]
@kezhenxu94 This is a modification of the e2e test case, please help check it.

**Comment by wu-sheng** [comment]
@dmsolr please compare this with your PR too.

File: test/e2e/e2e-profile/e2e-profile-test-runner/src/docker/clusterize.awk [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/usr/bin/awk -f
+
+BEGIN {
```
Comment (kezhenxu94) [inline test/e2e/e2e-profile/e2e-profile-test-runner/src/docker/clusterize.awk]: The file name `clusterize.awk` should be changed (after copying), `clusterize.awk` is originally used to modify the `application.yml` to make it run in cluster mode (to test coordinators), here should be named with something like `adapt_storage` or other storage related names, if you like

**Pull request merged**